### PR TITLE
QPD-1470: updated documentation as per the new changes

### PR DESCRIPTION
**Pull request by niharp2**
Updated readthedocs as per the new changes in implementation and use.

### CODE REVIEW

File: docs/source/_build/_sources/model_creation.rst.txt [hunk 1]
```diff
@@ -212,9 +212,76 @@ Example
 
    <div class="note">
 
-Note: 1. The model is expected to do the cleanup for the missing data in the window.
-2. Users are expected to pass data for window duration while experimenting, it is when once
-the model is deployed the window data will be fetched automatically.
+Note: 1. The predict function from model with window should return only one prediction at a time,
+considering that the input_df will only contain data for one datapoint with its respective previous data points for the specified duration.
+predict can return none, in case if there is any missing previous data.
```
Comment (jacob-issac) [inline docs/source/_build/_sources/model_creation.rst.txt:217]: "if there" can be removed

File: docs/source/_build/_sources/model_creation.rst.txt [hunk 2]
```diff
@@ -212,9 +212,76 @@ Example
...
+2. The window duration will be validated on the basis of the available resources in the provided ML_Node or best ML_Node.
+3. Users are expected to pass data for one datapoint along with its respective previous data in input_df while experimenting, it is when once
+the model is deployed the previous data will be fetched automatically for the streaming datapoints.
```
Comment (jacob-issac) [inline docs/source/_build/_sources/model_creation.rst.txt:220]: deployed,

File: docs/source/_build/model_creation.html [hunk 3]
```diff
@@ -176,9 +176,58 @@ <h3>Example<a class="headerlink" href="#id1" title="Permalink to this headline">
                    <span class="p">)</span>
 </pre></div>
 </div>
-<div class="note"><p>Note: 1. The model is expected to do the cleanup for the missing data in the window.
-2. Users are expected to pass data for window duration while experimenting, it is when once
-the model is deployed the window data will be fetched automatically.</p>
+<div class="note"><p>Note: 1. The predict function from model with window should return only one prediction at a time,
+considering that the input_df will only contain data for one datapoint with its respective previous data points for the specified duration.
+predict can return none, in case if there is any missing previous data.
```
Comment (jacob-issac) [inline docs/source/_build/model_creation.html:181]: looks like an incomplete sentence

File: docs/source/_build/model_creation.html [hunk 4]
```diff
@@ -176,9 +176,58 @@ <h3>Example<a class="headerlink" href="#id1" title="Permalink to this headline">
...
+2. The window duration will be validated on the basis of the available resources in the provided ML_Node or best ML_Node.
+3. Users are expected to pass data for one datapoint along with its respective previous data in input_df while experimenting, it is when once
```
Comment (jacob-issac) [inline docs/source/_build/model_creation.html:183]: "it is when" can be removed

File: docs/source/_build/model_creation.html [hunk 5]
```diff
@@ -176,9 +176,58 @@ <h3>Example<a class="headerlink" href="#id1" title="Permalink to this headline">
...
+the model is deployed the previous data will be fetched automatically for the streaming datapoints.
```
Comment (jacob-issac) [inline docs/source/_build/model_creation.html:184]: deployed,

**Review by jacob-issac — CHANGES_REQUESTED** [review]

**Review by jacob-issac — APPROVED** [review]

**Review by scriperdjq — APPROVED** [review]

**Pull request merged**